app/src/main/contrib/components/Responsibilities/Overview/ConfidentialityAgreement.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from "react";
+import {settings} from "../../../../shared/Settings";
+import {connect} from "react-redux";
+import {branch, compose, renderNothing} from "recompose";
+import {Dispatch} from "redux";
+import {ContribAppState} from "../../../reducers/contribAppReducers";
+import {default as withLifecycle, LifecycleMethods} from "@hocs/with-lifecycle";
+import {userActionCreators} from "../../../actions/userActionCreators";
+
+export interface ConfidentialityAgreementPublicProps {
+    touchstoneId: string;
+}
+
+export interface ConfidentialityAgreementProps extends ConfidentialityAgreementPublicProps {
+    signAgreement: () => void;
+    signed: boolean;
+    getConfidentiality: () => void;
+}
+
+const fullConfidentialityAgreement = require('./rfp-applicants-confidentiality.pdf');
+
+export const ConfidentialityAgreementComponent: React.SFC<ConfidentialityAgreementProps>
+    = (props: ConfidentialityAgreementProps) => {
+
+    if (settings.isApplicantTouchstone(props.touchstoneId)) {
```

Comment:
I think this check is redundant because of composing with `branch` below.

Reply:
Yes, good catch, thanks